src/Components/Table/Metadata.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import './Metadata.css'
+interface MetadataProps {
+  token: string;
+  route: string;
+  name: string;
+}
+
+interface MetadataState {
+  data: any;
+}
+
+
+/**
+ * Metadata component
+ */
+export default class Metadata extends React.Component<MetadataProps, MetadataState> {  
+  constructor(props: MetadataProps) {
+    super(props);
+    this.state = {
+      data: {recordHeader: [], records: [], totalCount: 0}
+    }
+  }
+
+  componentDidMount(){
+    let apiUrl = `${process.env.REACT_APP_DJLABBOOK_BACKEND_PREFIX}` + this.props.route;
```

Comment:
Update throughout the environment variable naming scheme.

```suggestion
    let apiUrl = `${process.env.REACT_APP_DJSCIVIZ_BACKEND_PREFIX}` + this.props.route;
```